src/LNEC_SAR_Bathymetry/Toolbox/CSAR_ImageProcessing.py

```diff
@@ -123,7 +123,7 @@ def InputSubsetParameters():
 	file_path_name = CL.File_path_name('', args.input, '', '')
 	Point = np.zeros(2) + 1000; Point = Point.astype(int)
 	LandMask_Parameters = CL.LandMaskParameters()	
-	ProcessingParameters = CL.Processing_Parameters('uint16', Slant_Correction, 1., Contrast_Stretch, LandMask_Parameters)
+	ProcessingParameters = CL.Processing_Parameters(Slant_Correction, 1., Contrast_Stretch, LandMask_Parameters)
```

Comment:
This change is not correct. The class constructor is receiving wrong parameters.